native/src/components/Note.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ReactElement } from 'react'
+import styled from 'styled-components/native'
+
+import { NoteIcon } from '../assets'
+import Icon from './base/Icon'
+
+const NoteBox = styled.View<{ visible: boolean }>`
+  background-color: ${props => props.theme.colors.themeColor};
+  margin-top: 12px;
+  padding: 12px;
+  flex-direction: row;
+  opacity: ${props => (props.visible ? '1' : '0')};
+`
+
+const NoteText = styled.Text`
+  font-family: ${props => props.theme.fonts.native.decorativeFontRegular};
+  font-size: 12px;
+  flex: 1;
+  flex-wrap: wrap;
+`
+
+const StyledIcon = styled(Icon)`
+  align-self: center;
+  margin-right: 12px;
+`
+
+type NoteProps = {
+  text: string
+  visible: boolean
+}
+
+const Note = ({ text, visible }: NoteProps): ReactElement => (
+  <NoteBox visible={visible} aria-hidden={!visible}>
```

Comment:
I'd prefer it if the button moves up when the note disappears. I also checked back with @hauf-toni regarding this. Why did you prefer hiding it?

Reply:
That was the way it was implemented bevor. It was at least partly on my initiative though since I don't like jumping layouts. If I want to click the button and then it moves up that is annoying to me. However, if UI/UX says thats how it should be done we can easily do that.